ERCS/erc-7776.md

```diff
@@ -0,0 +1,554 @@
+---
+eip: 7776
+title: Transparent Financial Statements
+description: A.P.I. for Protocols to Ensure Open Transparent Financial Data for all Investors.
+author: Ignacio Ceaglio (@Nachoxt17) <ignacioceaglio@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/erc-xxxx-transparent-financial-statements/21191
+status: Review
+type: Standards Track
+category: ERC
+created: 2024-09-20
+requires: 20
+---
+
+## Abstract
+
+This proposal defines a standard A.P.I. that enables E.V.M. Blockchain-based companies (or also called "Protocols") to publish their financial information, specifically Income Statements and Balance Sheets, on-chain in a transparent and accessible manner through Solidity Smart Contracts. This standard aims to emulate the reporting structure used by publicly traded companies in traditional stocks markets, like the S.E.C. 10-Q filings. The financial statements include key information, namely as Revenue, Cost of Goods Sold, Operating Expenses, Operating Income, Earnings before Interest, Taxes, Depreciation, and Amortization (E.B.I.T.D.A.) and
+Earnings Per Token (E.P.Share-Token), allowing investors to assess the financial health of blockchain-based companies in a standardized, transparent, clear and reliable format.
```

Comment:
The usual style for ERCs is to not use periods between letters in an initialism. Please make this change throughout the document. I won't be commenting on each line.

```suggestion
This proposal defines a standard API that enables EVM Blockchain-based companies (or also called "Protocols") to publish their financial information, specifically Income Statements and Balance Sheets, on-chain in a transparent and accessible manner through Solidity Smart Contracts. This standard aims to emulate the reporting structure used by publicly traded companies in traditional stocks markets, like the SEC 10-Q filings. The financial statements include key information, namely as Revenue, Cost of Goods Sold, Operating Expenses, Operating Income, Earnings before Interest, Taxes, Depreciation, and Amortization (EBITDA) and
Earnings Per Token (E.P.Share-Token), allowing investors to assess the financial health of blockchain-based companies in a standardized, transparent, clear and reliable format.
```

Reply:
Sure👍